### PR TITLE
Allow a config option for users in the beta group.

### PR DESCRIPTION
**Pull request by kadams54**

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6161/
Test FAILed.

**Comment by kadams54** [comment]
Not really sure what the best way is to QA is this, as it requires the beta flag and an uncommitted (new) model. Additionally, you need a way (e.g., modifying the settings .ini file) to control the config being passed into the GUI. The best I was able to do was to run the GUI in local dev/fakebackend, with a modified development.ini to add `jujugui.betaUser = false`. I then set a breakpoint in `app.js` and manually set the `modelCommitted` variable to false. The beta sign up screen was triggered. I repeated the same steps with `jujugui.betaUser = true` in development.ini and the beta sign up screen was not triggered.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6162/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6168/
Test PASSed.

**Comment by jaycee** [comment]
QA :ok: via QA in another branch.

File: jujugui/views.py [hunk 1]
```diff
@@ -133,7 +134,8 @@ def config(request):
         'socketTemplate': settings['jujugui.socketTemplate'],
         'gisf': settings['jujugui.gisf'],
         'staticURL': settings['jujugui.static_url'],
-        'gisfLogout': gisfLogout
+        'gisfLogout': gisfLogout,
+        'betaUser': settings.get('jujugui.betaUser', False),
```
Comment (frankban) [inline jujugui/views.py]: IIRC we have simple tests for every option, please take a look at test_views.

File: jujugui/views.py [hunk 2]
```diff
@@ -98,6 +98,7 @@ def config(request):
         'viewContainer': '#main',
         'baseUrl': base_url,
         'transitions': False,
+        # XXX kadams54: cachedFonts doesn't appear to be used anywhere.
```
Comment (frankban) [inline jujugui/views.py]: Let's kill it then please.

Comment (kadams54) [inline jujugui/views.py]: I could say the same about line 125 :wink:. It didn't appear to be used anywhere based on a quick grep through the code, but I'd want to do a more thorough investigation before removing.

File: jujugui/views.py (hunk 1 above)
Comment (kadams54) [inline jujugui/views.py]: Added assertions for this config.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6178/
Test PASSed.

**Comment by kadams54** [comment]
Thanks all! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6179/
Test PASSed.

**Pull request merged**